webalyzer/base/templates/home.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block body %}
+<h1>Webalyzer</h1>
+<h2>Collector Instructions</h2>
+<ol>
+    <li>Add the following to your page(s):
+<code>
+    &lt;script src="{{ request.scheme }}://{{ request.get_host }}/static/collect.js"
```

Comment:
Why not use `{{ request.build_absolute_uri }}{% static "collect.js" %}`?

Needs a `{% load staticfiles %}` above.